mockgen/reflect.go

```diff
@@ -41,8 +41,13 @@ func Reflect(importPath string, symbols []string) (*model.Package, error) {
 
 	progPath := *execOnly
 	if *execOnly == "" {
+		// Put program in working directory to pick up packages in vendor/ during build
+		wDir, err := os.Getwd()
```

Comment:
@dsymonds re-reading the [vendor spec](https://docs.google.com/document/d/1Bz5-UB7g2uPBdOx-rw5t9MxJwkfpx90cqG9AFL0JAYo/edit) I'm not sure there's an easy way to do that...the `vendor` directory could exist on a completely different subtree in the filesystem, so it's not as simple as looking for the existence of a `vendor` directory in the working directory.